aw_watcher_window/lib.py

```diff
@@ -1,19 +1,77 @@
 import sys
+import json
+import logging
 from typing import Optional
 
+logger = logging.getLogger(__name__)
 
-def get_current_window_linux() -> Optional[dict]:
-    from . import xlib
-    window = xlib.get_current_window()
 
-    if window is None:
-        cls = "unknown"
-        name = "unknown"
-    else:
-        cls = xlib.get_window_class(window)
-        name = xlib.get_window_name(window)
+class Linux:
```

Comment:
All other platform specific changes are in seperate files so would make more sense to create a new gnome_wayland.py file and only check for which backend to use in lib.py